### ISSUE TITLE
Automatic Cleanup continues to trigger every minute

### ISSUE THREAD

**Issue by awaldrop** [SEServerExtender/EssentialsPlugin#27]
![image](https://cloud.githubusercontent.com/assets/3308202/7324078/3f418eee-ea81-11e4-8256-9d82bb570ea2.png)

I have a basic 'no beacon' cleanup set to run at noon, with a 15,10,5,3,1 minute warning. Everything seems to work except after it triggers the cleanup, it continues to do it every minute until i disable the plugin.

Relevant Config XML:
```
  <CleanupTriggerItems />
  <CleanupTimedItems>
    <SettingsCleanupTimedItem>
      <Restart>2015-04-23T12:00:00</Restart>
      <NotificationItemsRan />
      <LastRan>2015-04-20T11:59:59.6958688-04:00</LastRan>
      <Enabled>true</Enabled>
      <RestartTime>12:00</RestartTime>
      <ScanCommand>excludesblocksubtype:Beacon:1</ScanCommand>
      <Reason>derelict ship cleanup</Reason>
    </SettingsCleanupTimedItem>
  </CleanupTimedItems>
  <CleanupNotificationItems>
    <SettingsCleanupNotificationItem>
      <lastRan>2015-04-20T11:45:00.5082101-04:00</lastRan>
      <Enabled>true</Enabled>
      <MinutesBeforeCleanup>15</MinutesBeforeCleanup>
      <Message>Server Cleanup in 15 Minutes!</Message>
    </SettingsCleanupNotificationItem>
    <SettingsCleanupNotificationItem>
      <lastRan>2015-04-20T11:50:00.4431072-04:00</lastRan>
      <Enabled>true</Enabled>
      <MinutesBeforeCleanup>10</MinutesBeforeCleanup>
      <Message>Server Cleanup in 10 Minutes!</Message>
    </SettingsCleanupNotificationItem>
    <SettingsCleanupNotificationItem>
      <lastRan>2015-04-20T11:55:00.6170193-04:00</lastRan>
      <Enabled>true</Enabled>
      <MinutesBeforeCleanup>5</MinutesBeforeCleanup>
      <Message>Server Cleanup in 5 Minutes!</Message>
    </SettingsCleanupNotificationItem>
    <SettingsCleanupNotificationItem>
      <lastRan>2015-04-20T11:57:00.7327872-04:00</lastRan>
      <Enabled>true</Enabled>
      <MinutesBeforeCleanup>3</MinutesBeforeCleanup>
      <Message>Server Cleanup in 3 Minutes!</Message>
    </SettingsCleanupNotificationItem>
    <SettingsCleanupNotificationItem>
      <lastRan>2015-04-20T11:59:00.9445593-04:00</lastRan>
      <Enabled>true</Enabled>
      <MinutesBeforeCleanup>1</MinutesBeforeCleanup>
      <Message>Final Warning: Server Cleanup imminent!</Message>
    </SettingsCleanupNotificationItem>
    <SettingsCleanupNotificationItem>
      <lastRan>2015-04-20T10:07:20.8810806-04:00</lastRan>
      <Enabled>true</Enabled>
      <MinutesBeforeCleanup>1</MinutesBeforeCleanup>
      <Message>Cleanup In Progress...</Message>
    </SettingsCleanupNotificationItem>
  </CleanupNotificationItems>
```
Essentials Version 1.12.0.2392

If you need anything else please let me know.

**Comment by dodexahedron** [comment]
Has it always behaved this way or is this new behavior?

**Comment by LordNex** [comment]
Been this way ever since trythis gave up that I remember.

**Comment by dodexahedron** [comment]
Can you pastebin your actual log file during one of the times this was happening?
Ideally, I'd like to see the beginning and some continuation of the bad state.

**Comment by awaldrop** [comment]
I'm assuming the Main log (SpaceEngineerDedicated_yyyymmdd_hhmmss.log)

http://pastebin.com/YF4jK2aM

I Grabbed it to reference the above screen shot, it starts Just prior (11:47) with the cleanup set for Noon (12:00) and goes for an extra hour (1:00)

The log doesn't show me anything useful but hopefully it helps. If this isn't the log you are looking for, where might I find the one you want?

**Comment by dodexahedron** [comment]
I mean the SESE log, which will be located in the DedicatedServer64/logs folder.

-----Original Message-----

**Comment by awaldrop** [comment]
Here is the Server Extender Log, I may need to change a configuration option to get you more detailed logs though...

http://pastebin.com/qAttSgEP

I lined it up to start with the auto-save in the screen shot above until I unloaded the plugin to make it stop cleaning. 

This is from the Plugin Log: (Nothing for a long time before)

2015-04-24 12:00:00.3002|Info|EssentialsPlugin.Utility.CubeGrids+<>c__DisplayClass70.<DeleteGrids>b__6f|Cleanup
2015-04-24 12:00:00.3002|Info|EssentialsPlugin.Utility.CubeGrids+<>c__DisplayClass70.<DeleteGrids>b__6f|Cleanup
2015-04-24 12:48:40.2686|Info|EssentialsPlugin.Essentials.Shutdown|Shutting down plugin: Dedicated Server Essentials - 1.12.0.2392
2015-04-24 12:48:42.0728|Debug|EssentialsPlugin.Essentials.InitWithPath|Initializing Essentials plugin at path C:\Users\Administrator.DTPC\AppData\Roaming\SpaceEngineersDedicated\Mods\Essentials\
2015-04-24 12:48:43.1748|Info|EssentialsPlugin.ProcessHandlers.ProcessRestart.StopAllShips|Stopping all ships
2015-04-24 12:48:43.1968|Info|EssentialsPlugin.ProcessHandlers.ProcessRestart.StopAllShips|0 ships have been stopped

**Comment by dodexahedron** [comment]
The problem is that you have both in-game autosave and SESE autosave enabled.
This WILL cause crashes if they save at the same time.
If you update to 417, SESE now enforces that only the SESE autosave can be enabled, so that crash should go away.

**Comment by dodexahedron** [comment]
This should be fixed in 1.12.3.2412
Re-open if there are further issues with it.
